use-cases/virgo/trainer.py

```diff
@@ -108,26 +110,85 @@ def create_model_loss_optimizer(self) -> None:
             self.model, self.optimizer, **distribute_kwargs
         )
 
+    def create_dataloaders(self, train_dataset: Dataset, validation_dataset: Dataset | None = None, test_dataset: Dataset | None = None) -> None:
+        """
+        Override the create_dataloaders function to use the custom_collate function. 
+        """
+        self.train_dataloader = self.strategy.create_dataloader(
+            dataset=train_dataset,
+            batch_size=self.config.batch_size,
```

Comment:
actually here you are using the batch size from the config, another reason to get it from there